client/webpack.common.config.js

```diff
@@ -22,4 +23,9 @@ module.exports = {
   module: {
     loaders: [],
   },
+  plugins: [
+    new webpack.ProvidePlugin({
```

Comment:
I really don't understand why there's no module import for fetch. I find having to understand how this globally imported really awkward.

 http://forum.railsonmaui.com/t/understanding-the-webpack-module-bundler/336

I'm sort of leaning toward using https://github.com/mzabriskie/axios as I this would provide a layer of abstraction on top of the client side native fetch (or https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest).